test/Microsoft.AspNet.Mvc.FunctionalTests/Compiler/Resources/MvcTagHelpersWebSite.Customer.Index.html

```diff
@@ -0,0 +1,42 @@
+﻿
```

Comment:
Actually, I'm using the different controller/action name pairs.....but..I'll extend the convention :-)